packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -632,4 +636,257 @@ describe('Table', () => {
             verifyRenderedData(simpleTableData);
         });
     });
+
+    describe('table column sizing', () => {
```

Comment:
This seems like a large enough set of tests that it might be worth moving them to their own file, especially since we'll probably have more tests once we have interactive column resizing.